onnxruntime/test/optimizer/graph_transform_test.cc

```diff
@@ -5031,6 +5031,24 @@ TEST_F(GraphTransformationTests, MatMulIntegerToFloatTest) {
   EXPECT_EQ(op_to_count["Add"], 1);
 }
 
+#ifdef USE_DML
+ TEST_F(GraphTransformationTests, MatMulIntegerToFloat16Test) {
+  constexpr const ORTCHAR_T* model_uri = MODEL_FOLDER "fusion/matmul_integer_to_float16_int8.onnx";
+  std::shared_ptr<Model> p_model;
```

Comment:
Thank you for adding this.
I guess the model name is incorrect because there is no model with this name under `onnxruntime/test/testdata/transform/fusion` directory.
Please let me know if I am missing something.

Reply:
yes, will move the model copy under this directory